bin/build-galeforce.sh

```diff
@@ -4,7 +4,8 @@ pushd $(dirname "$0")
 source ./common.sh
 
 BUSYBOX_URL="https://busybox.net/downloads/binaries/1.26.2-defconfig-multiarch/busybox-armv6l"
-DROPBEAR_URL="http://archive.raspbian.org/raspbian/pool/main/d/dropbear/dropbear-bin_2017.75-1_armhf.deb"
+DROPBEAR_URL="http://archive.raspbian.org/raspbian/pool/main/d/dropbear/dropbear-bin_2016.74-5%2Bdeb9u1_armhf.debc"
```

Comment:
Need to remove the "c" at the end of the link
http://archive.raspbian.org/raspbian/pool/main/d/dropbear/dropbear-bin_2016.74-5%2Bdeb9u1_armhf.deb